sunshine/confighttp.cpp

```diff
@@ -502,6 +512,56 @@ void savePin(resp_https_t response, req_https_t request) {
   }
 }
 
+void unpairAll(resp_https_t response, req_https_t request){
+  if(!authenticate(response, request)) return;
+  
+  print_req(request);
+
+  pt::ptree outputTree;
+
+  auto g = util::fail_guard([&]() {
+    std::ostringstream data;
+    pt::write_json(data, outputTree);
+    response->write(data.str());
+  });
+
+  try {
+    nvhttp::erase_all_clients();
```

Comment:
Functions written by us shouldn't throw exceptions, if it does, that's a bug.
Pass errors by `int` instead.